source/install/config-proxy-nginx.rst

```diff
@@ -0,0 +1,88 @@
+.. _config-proxy-nginx:
+
+Configuring NGINX as a proxy for Mattermost Server
+==================================================
+
+**To configure NGINX as a proxy**
+
+1. Log into the server that hosts NGINX and open a terminal window.
+2. Create a configuration file for Mattermost.
+
+  ``sudo touch /etc/nginx/sites-available/mattermost``
+
+2. Open the file ``/etc/nginx/sites-available/mattermost`` as root in a text editor and replace its contents, if any, with the following lines. Make sure that you use your own values for the Mattermost server IP address and FQDN for *server_name*.
```

Comment:
I updated the numbering. See commit https://github.com/JeffSchering/docs/commit/3b73c7bd9dffe1adbd82f29cba48710ffad64915

In this case server_name is not inline code, but the name of a parameter in a config file. I've made a note to update the style guide to make param names italic.